krux_boto/cli.py

```diff
@@ -21,16 +23,17 @@
 from krux_boto.boto import add_boto_cli_arguments, get_boto, get_boto3, NAME
 from krux_boto.util import RegionCode
 
-from krux_boto.version import VERSION
-
 
 class Application(krux.cli.Application):
     # XXX: Usually, a VERSION constant should be set in __init__.py and be imported.
     #      However, krux-boto adds some basic classes to __init__.py and importing VERSION constant here
-    #      causes a dependency circle. Thus, set VERSION constant in version.py and import it
-    _VERSION = VERSION
+    #      causes a dependency circle. Thus, set VERSION constant in version.json and import it
```

Comment:
I think this comment should still be here because it's still true for this case.